src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Node<K, V> {
```

Comment:
let's make Node inner static class of HashMap, since it shouldn't be used outside of HashMap